### PR TITLE
Fix windows-related error messsages.

### LINKED ISSUE #5383: Fix quoting for WSL-related error messages
The e2e tests in CI don't run for Windows, and WSL tests run differently on linux, so the changes in quoting missed a couple of spots.

### Rancher Desktop Version

1.10dev

### What operating system are you using?

Windows

### PR DESCRIPTION
**Pull request by ericpromislow**
Fixes #5383

Also the restart on Windows was broken because it was looking for a message bar caption that is used only for Lima.

### CODE REVIEW

File: e2e/utils/TestUtils.ts [hunk 1]
```diff
@@ -254,22 +254,32 @@ export async function tool(tool: string, ...args: string[]): Promise<string> {
   }
 }
 
-export async function waitForRestartVM(progressBar: Locator, options = { timeout: 10_000, interval: 200 }): Promise<void> {
+export async function waitForRestartVM(progressBar: Locator, options = { timeout: -1, interval: 200 }): Promise<void> {
```
Comment (adamkpickering) [inline e2e/utils/TestUtils.ts:257]: Why the change to the `timeout` default? I don't see any places that use `waitForRestartVM` that specify something other than the default for `timeout`. And we just set it to a new default below.

File: e2e/utils/TestUtils.ts [hunk 2]
```diff
@@ -254,22 +254,32 @@ export async function tool(tool: string, ...args: string[]): Promise<string> {
...
   const startTime = new Date().valueOf();
-  const endTime = startTime + options.timeout;
+  let endTime = startTime + options.timeout;
```
Comment (adamkpickering) [inline e2e/utils/TestUtils.ts:259]: With the new default for `timeout`, this sets `endTime` to one ms before `nowTime` at first. This isn't causing any problem, but it is bad practice IMO.

**Review by adamkpickering — CHANGES_REQUESTED** [review]
Just a minor comment.

File: e2e/utils/TestUtils.ts (hunk 1 above)
Comment (ericpromislow) [reply e2e/utils/TestUtils.ts:257]: Seeing how nothing passes `options` to `waitForRestartVM`, let's just drop the options argument.

File: e2e/utils/TestUtils.ts (hunk 2 above)
Comment (ericpromislow) [reply e2e/utils/TestUtils.ts:259]: Good catch. Fixed by dropping the `options` argument and setting `timeout` before this line.

**Review by adamkpickering — APPROVED** [review]

**Pull request merged**